packages/cloud_firestore/cloud_firestore/README.md

```diff
@@ -97,6 +97,20 @@ Firestore.instance.runTransaction((Transaction tx) async {
 });
 ```
 
+## Faster iOS builds
+
+Cloud Firestore has dependencies on iOS that lead to significant build times (see https://github.com/FirebaseExtended/flutterfire/issues/349 and https://github.com/firebase/firebase-ios-sdk/issues/4284). You can substantially reduce build times after the initial build using [incremental installation](https://medium.com/@corteggo/incremental-pod-installation-cocoapods-1-7-0-multiple-projects-302da0208af3).
+
+Add the following to your app's ios/Podfile:
```

Comment:
Could we give more context about where in the Podfile this should be added?

Reply:
Done